anchore_engine/clients/grype_wrapper.py

```diff
@@ -396,6 +402,21 @@ class GrypeVulnerabilityMetadata(Base, UtilMixin):
     description = Column(String)
     cvss_v2 = Column(String)
     cvss_v3 = Column(String)
+    vulnerability = relationship(
+        "GrypeVulnerability", back_populates="vulnerability_metadata"
+    )
+
+    @property
+    def deserialized_links(self):
+        return json.loads(self.links)
+
+    @property
+    def deserialized_cvss_v2(self):
+        return json.loads(self.cvss_v2)
+
+    @property
+    def deserialized_cvss_v3(self):
+        return json.loads(self.cvss_v3)
```

Comment:
Just an FYI that this is changing soon to have a unified CVSS key that points to a value of 0 or more CVSS scores including vendors. https://github.com/anchore/grype-db/compare/v2...grype-287

This change is explained better in this issue https://github.com/anchore/grype/issues/287

Reply:
Nice, thanks @alfredodeza